src/components/LocationPage/ChartBlock.tsx

```diff
@@ -86,4 +84,26 @@ function ChartBlock(props: {
   );
 }
 
+function renderDisclaimer(
+  region: Region,
+  metric: Metric,
+  provenance: Sources | undefined,
+) {
+  const metricDefinition = getMetricDefinition(metric);
+  const standardDisclaimer = metricDefinition.renderDisclaimer(
+    region,
+    provenance,
+  );
+
+  // Preface with any region override disclaimer text.
+  const overrideDisclaimer = getRegionMetricOverride(region, metric)
+    ?.disclaimer;
+  return (
+    <>
+      {overrideDisclaimer && <MarkdownContent source={overrideDisclaimer} />}
+      {standardDisclaimer}
+    </>
+  );
+}
```

Comment:
Should we move this to `metricDefinition` and call `metricDefinition.renderDisclaimer` instead?

Reply:
Since the code is the same for every metric, I'd rather not duplicate it across each one.  I could move it to `metric.tsx`, but I'm not sure that's much better.  Feels better to keep it close to where it's being used.